seed/views/portfoliomanager.py

```diff
@@ -164,10 +175,30 @@ def get_list_of_report_templates(self):
         if 'rows' not in template_object:
             raise PMExcept("Could not find rows key in template response; aborting.")
         templates = template_object["rows"]
+        template_response = []
         for t in templates:
+            t['z_seed_child_row'] = False
+            template_response.append(t)
             _log.debug("Found template,\n id=" + str(t["id"]) + "\n name=" + str(t["name"]))
-
-        return templates
+            if 'hasChildrenRows' in t and t['hasChildrenRows']:
```

Comment:
This section of code is where I check for the child data request rows and add them to the array being returned.  I tag them with a key so that we can check later to determine which type we are operating upon.